freqtrade/optimize/backtesting.py

```diff
@@ -1159,6 +1158,7 @@ def backtest(self, processed: Dict,
         while current_time <= end_date:
             open_trade_count_start = LocalTrade.bt_open_open_trade_count
             self.check_abort()
+            strategy_safe_wrapper(self.strategy.bot_loop_start, supress_error=True)()
```

Comment:
i think if we assume this callback to be called in backtesting over time - it might make sense to add a `current_time` argument to it (it's currently not needed as it's only called once in backtesting).

otherwise we'll know "it's been called 200 times" - but we won't know "it's now 2022-12-15 22:05:00" ...